core/utils/type-utils/src/main/java/datawave/data/type/ListType.java

```diff
@@ -39,6 +39,14 @@ public void setDelegateFromString(String in) {
         setNormalizedValue(in);
     }
     
+    /**
+     * Assumes that {@link #setDelegate(Comparable)} has been called
+     */
+    public void setNormalizedValueFromDelegate() {
+        this.normalizedValue = getDelegateAsString();
+        this.normalizedValues = normalizeToMany(getDelegateAsString());
```

Comment:
Minor suggestion, depending on what `getDelegateAsString()` executes (i.e. the type may keep repeated calls), using `normalizedValue` as the input would avoid second call to `getDelegateAsString()`.

```suggestion
        this.normalizedValues = normalizeToMany(this.normalizedValue);
```